cmd/baton-okta/main.go

```diff
@@ -39,8 +39,9 @@ func main() {
 
 func getConnector(ctx context.Context, cfg *config) (types.ConnectorServer, error) {
 	l := ctxzap.Extract(ctx)
-
-	cb, err := connector.New(ctx, cfg.Domain, cfg.ApiToken, cfg.SyncInactiveApps)
+	cb, err := connector.New(ctx, cfg.Domain, cfg.ApiToken,
+		cfg.OktaClientId, cfg.OktaPrivateKey, cfg.OktaPrivateKeyId,
+		cfg.SyncInactiveApps, cfg.OktaProvisioning)
```

Comment:
I think we should pass a struct in instead of a bunch of string arguments that could be accidentally out of order.